webapp/actions/global_actions.jsx

```diff
@@ -596,3 +596,18 @@ export function redirectUserToDefaultTeam() {
         browserHistory.push('/select_team');
     }
 }
+
+export function updateOpenGraphMetaData(url) {
```

Comment:
@enahum, I'll change it to `requestOpenGraphMetadata` and also change `MetaData` to `metadata` everywhere. The word 'update' actually meant 'request for update in store', but that is confusing.